robot_skills/src/robot_skills/world_model_ed.py

```diff
@@ -26,6 +26,7 @@
 from robot_skills.util.entity import from_entity_info
 from robot_skills.robot_part import RobotPart
 
+infradius = 200000
```

Comment:
Why this magic number?

Reply:
Because while there are symbols for infinity in both python and c++. I don't think it is possible to send it over a rosmessage

Reply:
Infiniti can be send in a ros msg.

Reply:
Fixed the magic number both here and in https://github.com/tue-robotics/ed/pull/94